test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -879,16 +879,38 @@ TEST_P(SslSocketTest, GetUriWithUriSan) {
                .setExpectedSerialNumber(TEST_SAN_URI_CERT_SERIAL));
 }
 
-TEST_P(SslSocketTest, IpSan) {
-  const std::string client_ctx_yaml =
-      fmt::format(R"EOF(
+// Verify that IP SANs work with an IPv4 address specified in the validation context.
+TEST_P(SslSocketTest, Ipv4San) {
+  const std::string client_ctx_yaml = R"EOF(
   common_tls_context:
     validation_context:
       trusted_ca:
-        filename: "{{{{ test_rundir }}}}/test/config/integration/certs/upstreamcacert.pem"
-      verify_subject_alt_name: {}
-)EOF",
-                  Network::Test::getLoopbackAddressString(GetParam()));
+        filename: "{{ test_rundir }}/test/config/integration/certs/upstreamcacert.pem"
+      verify_subject_alt_name: 127.0.0.1
```

Comment:
Nit: Those are supposed to be strings, so maybe quote them just to stay on the safe side?